pkgs/development/compilers/bs-platform/build-bs-platform.nix

```diff
@@ -30,9 +30,7 @@ stdenv.mkDerivation {
     cp ${custom-ninja}/bin/ninja vendor/ninja/snapshot/ninja.linux
   '';
 
-  configurePhase = ''
-    node scripts/ninja.js config
-  '';
+  dontConfigure = true;
```

Comment:
I tracked down the issue that @bobzhang referred to in [this comment](https://github.com/NixOS/nixpkgs/pull/77726#discussion_r373833252).

Apparently running `node scripts/ninja.js config` will cause the build script to build the development version.